angular-ngrx-scss/src/app/repository/components/repo-header/repo-heading/repo-heading.component.scss

```diff
@@ -0,0 +1,35 @@
+@use '../../../../shared/styles/variables.scss';
+
+.heading {
+  display: flex;
+  align-items: center;
+  gap: 0.5rem;
+
+  &__container {
```

Comment:
Nitpick: the "heading__container" is such a generic name for an element. I'd prefer something ".heading__path", ".heading__breadcrumbs" or something similar.